webpack.dev.js

```diff
@@ -4,4 +4,8 @@ const common = require('./webpack.common.js');
 module.exports = merge(common, {
   mode: 'development',
   devtool: 'inline-source-map',
+  output: {
+    filename: 'webpack-bundle.js',
+    path: path.resolve(__dirname, 'dist')
+  }
```

Comment:
How does adding the same output to `dev` and `prod` any different than keeping one output in `common`?

How does that make it "cache busting"? Could you please explain this to me!

Reply:
In dev mode webpack dev server keeps watch for file content change and reloads the page. so there is no need of adding contentHash in dev mode. while in prod mode I have added contentHash so whenever the content is changed filename is changed. so browser loads new file instead of cached one. so that newly deployed changes reflect for all users.

Reply:
Perfect.